examples/FaxLineCreate.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+require_once __DIR__ . "/vendor/autoload.php";
+
+$config = Dropbox\Sign\Configuration::getDefaultConfiguration();
+
+// Configure HTTP basic authorization: api_key
+$config->setUsername("YOUR_API_KEY");
+
+$faxLineApi = new Dropbox\Sign\Api\FaxLineApi($config);
+
+$data = new Dropbox\Sign\Model\FaxLineCreateRequest();
+$data->setAreaCode("209")
```

Comment:
```suggestion
$data->setAreaCode(209)
```

Expect area code to be `int`